src/components/Filters/Filters.tsx

```diff
@@ -1,22 +1,43 @@
-import React, { useState, useRef } from 'react';
+import React, { useState } from 'react';
 import styles from './Filters.module.css';
 import Button from '../Button';
-import { CheckboxField, FiltersDrawer } from '../FiltersDrawer/FiltersDrawer';
+import { FiltersCheckboxField } from '../FiltersCheckboxField/FiltersCheckboxField';
+import { FiltersDrawer } from '../FiltersDrawer/FiltersDrawer';
 import Icon from '../Icon';
 
 export type Props = {
   /**
-   * Checkbox fields that will be displayed in the filters drawer.
+   * Text to be placed in the button that activates the Filters Drawer
    */
-  checkboxFields: CheckboxField[];
+  triggerText?: string;
+  /**
+   * Button status variant that indicates if filters had been selected.
+   */
+  hasFilters?: boolean;
```

Comment:
Maybe this could be more specific and be something like `hasSelectedFilters` or something? I know that makes it a bit longer 🤷

Reply:
All good, changed to `hasSelectedFilters`